widget/radio_test.go

```diff
@@ -92,3 +92,12 @@ func TestRadio_Remove(t *testing.T) {
 	assert.Equal(t, 1, len(radio.Options))
 	assert.Equal(t, 1, len(Renderer(radio).(*radioRenderer).items))
 }
+
+func TestRadio_DuplicatedOptions(t *testing.T) {
+	radio := NewRadio([]string{"Hi", "Hi", "Hi", "Another", "Another"}, nil)
+
+	Refresh(radio)
```

Comment:
I don't think we should rely on this refresh call - there has been no specific change to a field so the developer would not know it is required. Perhaps the deduplication needs to be called from the constructor function as well?